tests/test_ci_app_host_home_isolation.py

```
@@ -0,0 +1,163 @@
+
+    for key in (
+        "HOME",
+        "CFFIXED_USER_HOME",
+        "XDG_CONFIG_HOME",
+        "CMUX_APP_HOST_EXPECTED_HOME",
+        "CMUX_APP_HOST_EXPECTED_XDG_CONFIG_HOME",
+    ):
+        if f'EnvironmentVariable key="{key}"' in UNIT_SCHEME:
+            raise SystemExit(
+                f"FAIL: cmux-unit scheme must not override TEST_RUNNER_{key}"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)